projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/StaticRoute.java

```diff
@@ -105,8 +99,8 @@ public int hashCode() {
     int result = 1;
     result = prime * result + _network.hashCode();
     result = prime * result + _administrativeCost;
-    result = prime * result + ((_nextHopInterface == null) ? 0 : _nextHopInterface.hashCode());
-    result = prime * result + ((_nextHopIp == null) ? 0 : _nextHopIp.hashCode());
+    result = prime * result + _nextHopInterface.hashCode();
```

Comment:
If nextHopInterface is UNSET_NEXT_HOP_INTERFACE (which now replaces null), the hashcode is not zero, this may change the result value. I am not sure if that is desired